### ISSUE TITLE
При решении задачи Classification результаты принимают вещественные значения.

### ISSUE THREAD

**Issue by MAGLeb** [nccr-itmo/FEDOT#234]
Решая задачу классификации, результат предсказания не целые числа.
```
task = 'classification'
composer_params = {
    'max_depth': 2,
    'max_arity': 2,
    'learning_time': 0.1 
}

model = Fedot(ml_task=task,
              composer_params=composer_params)
chain = model.fit(features=np.array(features_train),
                        target=np.array(target_train))
prediction = model.predict(features=np.array(features_valid))

array([9.2810631e-01, 2.1132191e-03, 3.3170132e-03, 9.9691188e-01,
       9.9891078e-01, 9.9755633e-01, 9.9896133e-01, 4.2323591e-04,
       9.9714440e-01, 9.9877244e-01], dtype=float32)
```

Также хочется обратить внимание на большой LOG при подборе композиций из моделей:
```
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: Invalid chain configuration: Chain has incorrect models positions. Continue.
Error in chain assessment during composition: Invalid chain configuration: Chain has incorrect models positions. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Best metric is -0.9916806666666667
Generation num: 0
max_depth: 2, no improvements: 0
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
Error in chain assessment during composition: This MLPClassifier estimator requires y to be passed, but the target y is None.. Continue.
spent time: 0.3 min
Best metric is -0.9916806666666667
Composition time: 0.267 min
Algorithm was terminated due to processing time limit
GP composition finished
Fit chain from scratch
```
Мне кажется некоторые строки не дожны выводиться. Тем более я не ставил параметр verbose.
Возможно стоит вынести в отдельный issues

**Comment by nicl-nno** [comment]
> Решая задачу классификации, результат предсказания не целые числа.

Это вероятности класса 1. Если нужны метки классов на выходе, то нужно указать output_mode='labels'.

**Comment by nicl-nno** [comment]
>Также хочется обратить внимание на большой LOG при подборе композиций из моделей:

Ну тут большество строчек связаны с какой-то ошибкой при обучении модели с MLPClassifier в составе - судя по всему, ему вместо target приходит None почему-то.

**Comment by MAGLeb** [comment]
> > Решая задачу классификации, результат предсказания не целые числа.
> 
> Это вероятности класса 1. Если нужны метки классов на выходе, то нужно указать output_mode='labels'.

В interface API вроде бы нет такого параметра.

**Comment by nicl-nno** [comment]
Для API наверное стоит реализовать классические методы predict - для меток/predict_proba - для вероятностей. Чтобы sklearn-пользователям было привычнее.

**Comment by MAGLeb** [comment]
> > Также хочется обратить внимание на большой LOG при подборе композиций из моделей:
> 
> Ну тут большество строчек связаны с какой-то ошибкой при обучении модели с MLPClassifier в составе - судя по всему, ему вместо target приходит None почему-то.

Видимо стоит разобраться почему возникают эти ошибки. И все-таки прошлый вопрос не отпал. Эти ошибки связаны с работой алгоритма и стоит ли их выводить пользователю?

**Comment by nicl-nno** [comment]
Да, наверное по умолчанию лучше скрывать, пока не включен verbose.

**Comment by MAGLeb** [comment]
Подведу итог:
- для API метода fit добавить параметр verbose, чтобы не видеть лишний вывод. 
- добавить метод классу API predict_proba. predict будет возвращать классы, predict_proba вероятности принадлежности классу.
- разобраться почему падают ошибки при работе композера (можно в отдельную issues вынести).
